gammapy/utils/regions.py

```diff
@@ -14,10 +14,12 @@
 """
 
 import operator
+import os
 import numpy as np
 from scipy.optimize import Bounds, minimize
 from astropy import units as u
 from astropy.coordinates import SkyCoord
+from astropy.table import Table
```

Comment:
```suggestion
from astropy.table import Table
from gammapy.utils.scripts import make_path
```
this function can deal with paths and env variables.